docs/Edit.md

```diff
@@ -7,7 +7,7 @@ title: "The Edit Component"
 
 The `<Edit>` component is the main component for edition pages. It fetches a record based on the URL, prepares a form submit handler, and renders the page title and actions. It is not responsible for rendering the actual form - that's the job of its child component (usually a form component, like [`<SimpleForm>`](./SimpleForm.md)). This form component uses its children ([`<Input>`](./Inputs.md) components) to render each form input.
 
-![post edition form](./img/edit-view.png)
```

Comment:
why not keep the picture in the 'Usage' section, just like you did for List?

Reply:
Because this (outdated) screenshot doesn't match the code in the usage section.